api/src/database/helpers/schema/types.ts

```diff
@@ -155,4 +160,12 @@ export abstract class SchemaHelper extends DatabaseHelper {
 	preprocessBindings(queryParams: Sql): Sql {
 		return queryParams;
 	}
+
+	addInnerSortFieldsToGroupBy(
+		_groupByFields: (string | Knex.Raw)[],
```

Comment:
I did not want to put the underscore there, just because then it shows up in the method usage descriptions. Fine to keep it like this, tho

Reply:
Updated it for consistency with similar cases and also to satisfy the linter. Would you prefer the parameters wrapped in an `options` field instead?

Reply:
Don't have a preference. If you think `options` is more appropriate, let's go for options

Reply:
As discussed over chat, I'm resolving this conversation for consistency with other similar occurrences.